### PR TITLE
storage_service: Ignore dropped table for repair_updater

### PR DESCRIPTION
**Pull request by asias**
In case a table is dropped, we should ignore it in the repair_updater, since we can not update off strategy trigger for a dropped table.

Refs #12373

### CODE REVIEW

File: service/storage_service.cc [hunk 1]
```diff
@@ -2521,10 +2521,16 @@ future<node_ops_cmd_response> storage_service::node_ops_cmd_handler(gms::inet_ad
...
-                            ops_uuid, table.schema()->ks_name(), table.schema()->cf_name(), coordinator);
+                    try {
+                        auto& table = db.find_column_family(table_id);
+                        table.update_off_strategy_trigger();
+                        slogger.debug("repair[{}]: Updated off_strategy_trigger for table {}.{} by node {}",
+                                ops_uuid, table.schema()->ks_name(), table.schema()->cf_name(), coordinator);
+                    } catch (replica::no_such_column_family&) {
+                        // The table could be dropped by user, ignore it.
+                    } catch (...) {
+                        throw;
```
Comment (bhalevy) [inline service/storage_service.cc:2532]: why is that needed BTW?
this is the default behavior for any non-caught exception, no?

Comment (asias) [reply service/storage_service.cc:2532]: Yes. We could drop skip the default catch to save a few lines.
